code/datums/mutations/body.dm

```diff
@@ -375,3 +375,69 @@
 			owner.SetStun(owner.AmountStun()*2)
 			owner.visible_message("<span class='danger'>[owner] tries to stand up, but trips!</span>", "<span class='userdanger'>You trip over your own feet!</span>")
 			stun_cooldown = world.time + 300
+
+/datum/mutation/human/strongwings
+	name = "Strengthened Wings"
+	desc = "Subject's wing muscle volume rapidly increases."
+	quality = POSITIVE
+	difficulty = 12
+	instability = 15
+	species_allowed = list(SPECIES_APID, SPECIES_MOTH)
+
+/datum/mutation/human/strongwings/on_acquiring()
+	if(..())
+		return
+	var/obj/item/organ/wings/bee/beewings = locate(/obj/item/organ/wings/bee) in owner.internal_organs
+	var/obj/item/organ/wings/moth/mothwings = locate(/obj/item/organ/wings/moth) in owner.internal_organs
```

Comment:
you can only have one at a time, locate one check if it exists and then try locating second type (if first fails)